src/arrayReverse.test.js

```diff
@@ -7,14 +7,29 @@ describe(`Function 'arrayReverse':`, () => {
     expect(arrayReverse).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
+  it(`should return an array of reversed strings in reversed order`, () => {
+    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
```

Comment:
The expected result should be `['ymedacA', 'etaM']` to reflect both the reversed strings and their order.